src/v3/src/components/ReminderPrompt/ReminderPrompt.tsx

```diff
@@ -119,6 +120,7 @@ const ReminderPrompt: UISchemaElementComponent<{
         <TextWithActionLink
           uischema={{
             type: 'TextWithActionLink',
+            parserOptions: uischema.parserOptions,
```

Comment:
Do we always want a monochrome link in `ReminderPrompt`? If so, we could hard-code that here instead of pulling from `uischema`

Reply:
Ah, so you mean call the function here to build the parser options (for the link replacer) instead of doing it at the transformer layer. I like it, it removes duplicated effort! 🙏